solidity/migrations/2_deploy_contracts.js

```diff
@@ -80,6 +82,27 @@ module.exports = async function(deployer, network) {
   await deployer.link(Reimbursements, KeepRandomBeaconOperator);
   await deployer.link(BLS, KeepRandomBeaconOperator);
 
+  if(network === 'local') {
+    const initializationPeriod = 120;
+    const undelegationPeriod = 120
+    
+    // Let's deploy the old `TokenStaking` contract and deploy `StakingPortBacker`
```

Comment:
I think this comment can be skipped. From the code below it is very clear what contracts we want to deploy.